.coveralls.yml

```diff
@@ -0,0 +1,2 @@
+service_name: github-actions
+repo_token: 4K7LtmFlYnTWaMYypWe8fy6ruMnAfT1UG
```

Comment:
Is this ok to be public?

Reply:
Nope. If we are still sending this through Travis CI, the `repo_token` entry can be removed entirely. Otherwise, the Coveralls action on GH Actions should be used.

Reply:
I will look into the action. FYI, the public tokens are in the orchestra CI's, (link)[https://github.com/uavaustin/orchestra/blob/e30965fd71b0f9e4de82d0f8a5cef6770b60a4db/.travis.yml#L45]

Reply:
Those tokens in the `.travis.yml` file are encrypted; when Travis CI performs the build, it decrypts the tokens based on a secret token only known within the build environment.

Reply:
Got it! Thanks!

Reply:
I think I will just take out the coveralls for now since the unit tests on tfm are really minimal. Perhaps could add more comprehensive tests later..